pallets/dex-api/src/lib.rs

```diff
@@ -418,19 +413,12 @@ pub mod pallet {
     pub type EnabledSourceTypes<T: Config> = StorageValue<_, Vec<LiquiditySourceType>, ValueQuery>;
 
     #[pallet::genesis_config]
+    #[derive(Default)]
     pub struct GenesisConfig {
         pub source_types: Vec<LiquiditySourceType>,
     }
 
     #[cfg(feature = "std")]
```

Comment:
Is it ok to leave `#[cfg(feature = "std")]` for GenesisConfig?

Reply:
fixed it, thx